.github/workflows/python-publish.yml

```diff
@@ -0,0 +1,31 @@
+# This workflows will upload a Python Package using Twine when a release is created
```

Comment:
What about the licensing of such files?

Reply:
No idea. I just used a template file as-is, available from https://github.com/openvinotoolkit/nncf_pytorch/actions. Should we just slap an Apache header onto this file?

Reply:
Probably yes, I actually meant this.

Reply:
Done